.github/workflows/release.yaml

```diff
@@ -80,7 +80,38 @@ jobs:
           }
         shell: Rscript {0}
 
-      - name: Release
+      - name: cross compile rust object files for another target arch
+        if: matrix.config.target != ''
+        run: |
+          pwd
+          make requirements-rs
+          rustup target add "$RUNNER_TARGET"
+          BUILD_COMMAND="cargo build --lib --profile ${RPOLARS_PROFILE}  --target "$RUNNER_TARGET" --manifest-path=./src/rust/Cargo.toml"
+          if [ "${RPOLARS_FULL_FEATURES}" == 'true' ]
+          then
+            ${BUILD_COMMAND} --features "full_features"
+          else
+            ${BUILD_COMMAND}
+          fi
```

Comment:
I is copied from the Makevars file I think . It would be possible to rework the Makevars files to use more make calls more and thereby also call more make commands from the yaml workflows. But it is tedious as the iteration cycle for me 1.5 hour compilation. As I only work on rpolars ~3 hours a week at the moment,  it would delay this PR weeks I think. I don't think the current status looks bad at all.

Reply:
Sorry I have not gotten around to this. I have time today to complete this PR.